liveblog.php

```diff
@@ -734,7 +734,7 @@ public static function admin_enqueue_scripts( $hook_suffix ) {
 			$use_rest_api = 0;
 
 			if ( self::use_rest_api() ) {
-				$endpoint_url = trailingslashit( trailingslashit( trailingslashit( WPCOM_Liveblog_Rest_Api::build_endpoint_base() ) . $post->ID ) . 'post_state' );
+				$endpoint_url = WPCOM_Liveblog_Rest_Api::build_endpoint_base() . $post->ID . '/' . 'post_state';
```

Comment:
Yeah, that underscore doesn't make much sense with the rest of the URL structure.
I think just `state` would work fine here.